run

```diff
@@ -486,6 +494,14 @@ copy_nvjpeg_files_() {
     local root_folder=${1:-${TOP}}
     local arch_name="$(get_arch_name_)"
     local nvjpeg_search="${root_folder}/temp/cuda/include:${root_folder}/temp/cuda/lib64"
+    if [[ ! -z "${CONDA_PREFIX+x}" ]]
+    then
+      nvjpeg_search="${CONDA_PREFIX:-}/include:${CONDA_PREFIX:-}/lib ${nvjpeg_search}"
+    fi
+    if [[ ! -z "${PREFIX+x}" ]]
+    then
+      nvjpeg_search="${PREFIX:-}/include:${PREFIX:-}/lib ${nvjpeg_search}"
+    fi
     local cuda_version="11.6"
     local nvjpeg_version="11-6_11.6.0.55-1"
```

Comment:
This is hard-coding 11.6, which needs to be fixed for CUDA 12

Reply:
Actually this can be ignored since it is only used as a fallback when nvJPEG (of any version) cannot be found. So it doesn't matter here